RubberduckTests/Refactoring/EncapsulateField/EncapsulateFieldTestComponentResolver.cs

```diff
@@ -0,0 +1,147 @@
+﻿using Rubberduck.Parsing.Rewriter;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Refactorings;
+using Rubberduck.Refactorings.CreateUDTMember;
+using Rubberduck.Refactorings.EncapsulateField;
+using Rubberduck.Refactorings.ReplaceReferences;
+using Rubberduck.Refactorings.ReplacePrivateUDTMemberReferences;
+using Rubberduck.Refactorings.ReplaceDeclarationIdentifier;
+using Rubberduck.Refactorings.EncapsulateFieldUseBackingUDTMember;
+using Rubberduck.Refactorings.EncapsulateFieldUseBackingField;
+using Rubberduck.Refactorings.EncapsulateFieldInsertNewCode;
+using System;
+
+namespace RubberduckTests.Refactoring.EncapsulateField
+{
+    public class EncapsulateFieldTestComponentResolver
```

Comment:
Is this to work with the lack of CW? Consider using a stripped down CW for the tests? That would give a better fidelity compare the actual code and the unit tests.

Reply:
Yes...this class/approach helped me to manage and evaluate my ctor injected dependencies, understand a bit more about what CW provides, and made object instantiations for testing quite a bit easier.  That said, I never considered that I could/should use a stripped down version of CW for the tests.  I'll check into it - and it will be, no doubt, a learning experience.  TBH, CW is still quite mysterious to me.

Reply:
There is [an example](https://github.com/rubberduck-vba/Rubberduck/tree/next/RubberduckTests/Refactoring/MockIoC), inspired by [this article](https://blog.ploeh.dk/2013/03/11/auto-mocking-container/). HTH!

Reply:
Created a `WindsorContainer` that managed dependencies for the tests.  

I can see (now in retrospect) that the real value of a container to support testing is highest if it exists right from the start.  Since design/development introduces a lot of churn to various constructor signatures...the test code would have simply continued to compile without modification after changes to the refactoring's constructors (which was frequent when refactoring to use DI).  I think a container to support testing would be overkill in most cases - but I think it is well worth having for this refactoring.